app/javascript/src/datatable.js

```diff
@@ -0,0 +1,24 @@
+// This makes sure Turbolinks doesn't double load the datatable code
+// if you press the browser back button
+let dataTable = ""
+
+document.addEventListener("turbolinks:before-cache", function () {
+  if (dataTable !== null) {
+      dataTable.destroy();
+      dataTable = null;
+  }
+});
+
+document.addEventListener('turbolinks:load', function () {
+  dataTable = $('#datatable').DataTable({
+    "processing": true,
+    "serverSide": true,
+    "pageLength": 15,
```

Comment:
Will this be configurable?

Reply:
there's a little dropdown on the datatable to pick the number per page if that counts :)  but yes it could be set per-page if we need?